cit3.0-web/src/components/SearchFlyoutContent/SearchFlyoutContent.js

```diff
@@ -276,8 +253,8 @@ export default function SearchFlyoutContent({ setQuery }) {
       query.append("zoning", zoningQueryFilters);
     }
 
-    if (connectivityIsSelected) {
-      query.append("connectivity", connectivityQueryFilters);
+    if (connectivitySwitchValue) {
+      query.append("connectivity", connectivitySwitchValue ? "Y" : "");
```

Comment:
For consistency this should maybe be "N" for the switch being off, instead of "". It should work either way though.
```suggestion
      query.append("connectivity", connectivitySwitchValue ? "Y" : "N");
```